console-util/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+name = "console-util"
+version = "0.1.0"
+authors = ["Eliza Weisman <eliza@buoyant.io>"]
+edition = "2018"
```

Comment:
hmm, i'm not the biggest fan of creating a separate `console-util` crate just for the duration-parsing code...but I agree that it doesn't make sense in `console-api`. i think it might be better if it went in a crate specifically for duration parsing that we could publish to crates.io eventually.

alternatively, we might just want to replace our homemade duration parser with an existing library, especially if an existing library would accept more durations we don't handle nicely at the moment. i would probably go with [the `humantime` crate](https://lib.rs/crates/humantime) for this. i believe [`chrono`](https://lib.rs/crates/chrono) can also do this, but i think `humantime` would be a better choice for us, since `chrono` includes a lot of code for working with _dates_, which we don't need. `humantime` (45KB / 917 SLoC) is a significantly smaller dependency than `chrono` (760KB / 11k SLoC), and _only_ provides code for parsing/formatting durations. i think maybe we should switch to `humantime`.